lib/elixir_analyzer/exercise_test/common_checks/uncommon_errors.ex

```diff
@@ -0,0 +1,19 @@
+defmodule ElixirAnalyzer.ExerciseTest.CommonChecks.UncommonErrors do
+  @moduledoc """
+  This is an exercise analyzer extension module used for common tests looking for uncommon errors raised
+  """
+
+  defmacro __using__(_opts) do
+    quote do
+      feature "raises function clause error" do
+        find :none
+        type :actionable
+        comment ElixirAnalyzer.Constants.solution_raise_fn_clause_error()
+
+        form do
+          raise FunctionClauseError
+        end
```

Comment:
Any other forms that need to be handled?

Reply:
One more. `raise` can also be used with structs:
```elixir
iex(6)> raise %FunctionClauseError{}
** (FunctionClauseError) no function clause matches
```